src/core/tests/type_prop/embeddingbag_offsetssum.cpp

```diff
@@ -231,6 +266,24 @@ TEST(type_prop, ebos_fail_indices_1d) {
     }
 }
 
+TEST(type_prop, ebos_fail_emb_table_0d) {
+    auto emb_table = make_shared<op::Parameter>(element::f32, Shape{});
+    auto indices = make_shared<op::Parameter>(element::i64, Shape{4});
+    auto offsets = make_shared<op::Parameter>(element::i64, Shape{3});
+    auto per_sample_weights = make_shared<op::Parameter>(element::f32, Shape{4});
+    auto default_index = make_shared<op::Parameter>(element::i64, Shape{});
+
+    try {
```

Comment:
Detail: include common assertion utils and use `OV_EXPECT_THROW` here?

Reply:
Done